.gitignore

```diff
@@ -1,6 +1,7 @@
 # Environment vars
 .env
 env.yaml
+.vscode
```

Comment:
Please move this to the bottom of `.gitignore` (currently it's grouped with "environment vars" ignored files)